inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -86,49 +87,45 @@ def extract_journal_info(obj, eng):
 
 def extract_references(filepath):
     """Extract references from PDF and return in INSPIRE format."""
+
+    def field_builder(builder):
+        def _build_field(value_l):
+            for value in value_l:
+                builder(value)
+        return _build_field
+
     references = extract_references_from_file(
         filepath,
         reference_format=u"{title},{volume},{page}",
     )
     mapped_references = []
-    if references.get('references'):
-        for ref in references.get('references'):
-            reference = {
-                'curated_relation': False,
-                'raw_refs': [
-                    {
-                        'position': '',
-                        'schema': '',
-                        'source': '',
-                        'value': json.dumps(ref),
-                    }
-                ],
-                'record': {
-                    '$ref': ''
-                },
-                'reference': {
-                    'authors': ref.get('author'),
-                    'book_series': ref.get('book_series'),
-                    'collaboration': ref.get('collaboration'),
-                    'dois': ref.get('dois'),
-                    'imprint': ref.get('imprint'),
-                    'misc': ref.get('misc'),
-                    'number': ref.get('linemarker'),
-                    'persistent_identifiers': ref.get(
-                        'persistent_identifiers'
-                    ),
-                    'publication_info': {
-                        'year': ref.get('year'),
-                        'journal_title': ref.get('journal_title'),
-                        'journal_volume': ref.get('journal_volume'),
-                        'page_start': ref.get('journal_page'),
-                        'artid': ref.get('journal_reference'),
-                    },
-                    'texkey': ref.get('texkey'),
-                    'titles': ref.get('titles'),
-                    'urls': ref.get('urls'),
-                },
-            }
-            mapped_references.append(reference)
+
+    # TODO: these keys are returned but we don't need them .
+    # excluded_keys = ['linemarker', 'journal_volume', 'journal_title', 'journal_page']
```

Comment:
That's not true. `linemarker` should go to [`label`](https://github.com/inspirehep/inspire-schemas/blob/master/inspire_schemas/records/elements/reference.yml#L94). The others are redundant with `journal_reference` indeed.